packages/apps/explorer/README.md

```diff
@@ -0,0 +1,103 @@
+# Kadena Block Explorer Web
+
+## Getting Started
+
+First, install the dependencies:
+```bash
+npm install
+# or
+yarn install
+
```

Comment:
- Modify to be rush compatible (eg `rush install --to @kadena/block-explorer`)
- Remove `yarn` commands
- Replace `npx` with `pnpx`